index.html

```diff
@@ -18,11 +18,14 @@
     <!-- Bootstrap css -->
     <link rel="stylesheet" href="css/bootstrap.min.css">
     <link rel="stylesheet" href="css/bootstrap-reboot.min.css">
-    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css"/>
+    <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" />
+
+    <link rel="stylesheet" href="https://api.mapbox.com/mapbox-gl-js/v3.12.0/mapbox-gl.css" />
     <link rel="stylesheet" href="css/custom.css"/>
 
     <!-- External scripts -->
-    <script src="lib/bootstrap.bundle.min.js"></script>
+    <script src="js/bootstrap.bundle.min.js"></script>
+    <script src='https://api.mapbox.com/mapbox-gl-js/v3.12.0/mapbox-gl.js'></script>
```

Comment:
I'm currently using a cdn because I had trouble importing an installed version of mapbox gl js into the map script. There was an error about it not being a module.

I imagine that using jekyll would've been the answer there? I haven't made a jekyll site from scratch before so I didn't think of it at the time!

Reply:
it shouldn't require jekyll or any other service. have you tried saving https://api.mapbox.com/mapbox-gl-js/v3.12.0/mapbox-gl.js locally?

I have a local copy of maplibre working on https://ilsolarmap.com/